packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -124,6 +124,17 @@ In scenarios where there is a requirement for the card to fill the height of the
   code={FillHeightExample}
 />
 
+### Fill width
+
+Card width should fill the available space of the parent container. However, when wrapped in a flex container, the Card will shrink to fit the available space. To prevent this, you can use the `fillWidth` prop.
```

Comment:
```suggestion
By default, Card width fills the available space of the parent container. However, when wrapped in a flexbox, Card shrinks to fit the available space. To prevent this, use the `fillWidth` prop.
```